### ISSUE TITLE
Unlock wallet

### ISSUE THREAD

**Issue by 2ge** [syscoin/blockmarket-desktop-public#370]
Using BM3, there are some actions where wallet needs to be unlocked. Either user doesn't receive message (unlock wallet first), or it receives some other message (error). I am suggesting, action, where there is need to unlock wallet first should pop dialog with error message "This action needs unlocked walled" with two buttons "Unlock Wallet", "Cancel"

**Comment by jnessbcf** [comment]
Agreed with you.  We're going to be looking at wallet locking behavior holistically in a future release.
